tests/golem/task/test_requestedtaskmanager.py

```diff
@@ -0,0 +1,335 @@
+import asyncio
+
+from freezegun import freeze_time
+from golem_task_api.client import RequestorAppClient
+from golem_task_api.structs import Subtask
+from mock import Mock, patch
+import pytest
+from twisted.internet.defer import inlineCallbacks
+from twisted.trial.unittest import TestCase as TwistedTestCase
+
+from golem.app_manager import AppManager
+from golem.core.common import install_reactor
+from golem.tools.testwithreactor import uninstall_reactor
+from golem.core.deferred import deferred_from_future
+from golem.model import default_now, RequestedTask, RequestedSubtask
+from golem.task.envmanager import EnvironmentManager
+from golem.task.requestedtaskmanager import (
+    ComputingNode,
+    CreateTaskParams,
+    RequestedTaskManager,
+    ComputingNodeDefinition,
+)
+from golem.task.taskstate import TaskStatus, SubtaskStatus
+from golem.testutils import DatabaseFixture
+
+
+class TestRequestedTaskManager(DatabaseFixture, TwistedTestCase):
+    @classmethod
+    def setUpClass(cls):
+        try:
+            uninstall_reactor()  # Because other tests don't clean up
+        except AttributeError:
+            pass
+        cls.loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(cls.loop)
+        install_reactor()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        uninstall_reactor()
+        asyncio.set_event_loop(None)
+
+    def setUp(self):
+        super().setUp()
+        self.env_manager = Mock(spec=EnvironmentManager)
+        self.app_manager = Mock(spec=AppManager)
+        self.public_key = str.encode('0xdeadbeef')
+        self.rtm = RequestedTaskManager(
+            env_manager=self.env_manager,
+            app_manager=self.app_manager,
+            public_key=self.public_key,
+            root_path=self.new_path
+        )
+
+    def tearDown(self):
+        super().tearDown()
+
+    def test_create_task(self):
+        with freeze_time() as freezer:
+            # given
+            golem_params = self._build_golem_params()
+            app_params = {}
+            # when
+            task_id = self.rtm.create_task(golem_params, app_params)
+            freezer.tick()
+            # then
+            row = RequestedTask.get(RequestedTask.task_id == task_id)
+            assert row.status == TaskStatus.creating
+            assert row.start_time < default_now()
+            assert (self.new_path / golem_params.app_id / task_id).exists()
+
+    @inlineCallbacks
+    def test_init_task(self):
+        # given
+        mock_client = self._mock_client_create()
+        self._add_create_task_to_client_mock(mock_client)
+
+        task_id = self._create_task()
+        # when
+        yield self._coro_to_def(self.rtm.init_task(task_id))
+        row = RequestedTask.get(RequestedTask.task_id == task_id)
+        # then
+        assert row.status == TaskStatus.creating
+        assert mock_client.create_task.called_once_with(
```

Comment:
```suggestion
        mock_client.create_task.assert_called_once_with(
```
And the warning disappears (I dunno why, tho).

Reply:
thanks, that solved it :) merged your branch into this one